File: 프로그래머스/1단계/2016년.py
import datetime


def solution(a, b):
    weekend = ['MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT', 'SUN']

    return weekend[datetime.date(2016, a, b).weekday()]


solution(5, 24)  # TUE

# https://programmers.co.kr/learn/courses/30/lessons/12901?language=javascript

File: 프로그래머스/1단계/2016년.js
function solution(a, b) {
  const weekend = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"];
  const day = new Date(`2016-${a}-${b}`).getDay();

  return weekend[day];
}

solution(5, 24); // TUE

// https://programmers.co.kr/learn/courses/30/lessons/12901?language=javascript
